phoenix-core/src/main/java/org/apache/phoenix/util/ByteUtil.java

```diff
@@ -56,7 +56,34 @@ public class ByteUtil {
             EMPTY_BYTE_ARRAY);
     public static final ImmutableBytesWritable EMPTY_IMMUTABLE_BYTE_ARRAY = new ImmutableBytesWritable(
             EMPTY_BYTE_ARRAY);
-    
+
+
+    /** Mask for bit 0 of a byte. */
+    private static final int BIT_0 = 1;
```

Comment:
Why don't we have 0x01 here?

Reply:
Becuse that's how it's written in commons-codec where I copied this from.

But you're right, it's more consistent to use 0x01 here.

Reply:
Fixed.